DOCS.md

```diff
@@ -8,20 +8,217 @@ Registry: https://hub.docker.com/r/target/vela-terraform
 
 ## Usage
 
-COMING SOON!
+**NOTE: by default Terraform runs in the current directory. Use `directory: path/to/tf/files` to point Terraform at a file or files.**
+
+Sample of applying Terraform configuration:
+
+```yaml
+- name: apply
+  image: target/vela-terraform:latest
+  pull: true
+  parameters:
+    action: apply
+    auto_approve: true # Required for versions of Terraform 0.12.x
+```
+
+Sample of destroying Terraform configuration:
+
+```yaml
+- name: destroy
+  image: target/vela-terraform:latest
+  pull: true
+  parameters:
+    action: destroy
+    auto_approve: true # Required for versions of Terraform 0.12.x
+```
+
+Sample of adding init options to Terraform configuration:
```

Comment:
is the content for this missing intentionally? if so, maybe add `COMING SOON` treatment as done elsewhere